apps/api-runtime/src/main/java/com/akto/runtime/APICatalogSync.java

```diff
@@ -1206,89 +1206,98 @@ public void buildFromDB(boolean calcDiff, boolean fetchAllSTI) {
         }
     }
 
+    private static void build(SingleTypeInfo param, Map<Integer, APICatalog> ret) {
```

Comment:
rename the function to `buildHelper` please ?

Reply:
done